chipy_org/apps/sponsors/models.py

```diff
@@ -61,3 +61,6 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse("sponsor_detail", args=[self.slug])
+
+    class Meta:
+        ordering = ["name"]
```

Comment:
from pylint:

************* Module chipy_org.apps.sponsors.models
C: 66, 0: Final newline missing (missing-final-newline)
C: 66, 0: Mixed line endings LF and CRLF (mixed-line-endings)